doc/api/deprecations.md

```diff
@@ -3,9 +3,11 @@
 <!--introduced_in=v7.7.0-->
 <!-- type=misc -->
 
-Node.js may deprecate APIs when either: (a) use of the API is considered to be
-unsafe, (b) an improved alternative API is available, or (c) breaking changes to
-the API are expected in a future major release.
+Node.js may deprecate APIs in the following cases:
+
+* use of the API is considered to be unsafe
+* an improved alternative API is available
+* breaking changes to the API are expected in a future major release.
```

Comment:
Maybe we need to uppercase the first letter ?

Reply:
> Maybe we need to uppercase the first letter ?

Could go either way, I think, but I also mildly prefer a capital letter at the start for each of these. Each one can stand alone as a sentence and have a period at the end.

Reply:
Done